### PR TITLE
feat(css): add basic guideline for CSS

### PR DESCRIPTION
**Pull request by theetrain**
### Outline

- Add basic CSS guide.
- Add heading in root README for 'Content' as in 'Content Strategy'

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,10 @@ The following sections and placeholder articles are the suggested starting point
 
 _TBD_
 
+### Content
+
+_TBD_
```
Comment (ahmadnassri) [inline README.md]: lets not add any more TBDs, add content, otherwise don't change

File: development/css.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# CSS
+
+## Why
+
+CSS is an interesting language for having the perception of being easy to learn, yet difficult to scale and modularize. Front end teams often re-use the same CSS and want to be able to revisit a codebase months later to easily make changes.
```
Comment (ahmadnassri) [inline development/css.md]: the "why" needs to be in context of TELUS digital ... here are some questions to help you better phrase this:

- why are we using CSS directly (as oppose to some other visual language)?
- why are we writing CSS (as oppose to a transpiled alternative)?

File: development/css.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+## What
+
+CSS is Cascading Style Sheets. At a glance:
```
Comment (ahmadnassri) [inline development/css.md]: "what" in the context of TELUS Digital. not a global definition. for that use references.

File: development/css.md (hunk 2 above)
Comment (theetrain) [inline development/css.md]: Though I have a statement under 'how' that encourages SCSS, I'll include something here as well.

File: development/css.md (hunk 3 above)
Comment (theetrain) [inline development/css.md]: I was using this as an example https://github.com/telusdigital/reference-architecture/blob/master/development/javascript.md#what which appears to have a more objective 'what' than a Telus 'what'. Can you help provide more insight?

**Comment by theetrain** [comment]
@ahmadnassri updated.

File: development/css.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# CSS
+
+## Why
+
+Though every Telus web property uses CSS, developers rarely write CSS directly. CSS is often either included as part of [TDS components](http://tds.telus.com/), transpiled from SCSS, or embedded in [JS](./javascript.md) components.
+
+## What
+
+Every front end site uses CSS in some shape or form at Telus.
```
Comment (ahmadnassri) [inline development/css.md]: hint at better question:

> "what is the "CSS" entity here represented by in Telus?"

hint at answer:

> - the TDS compiled css
> - the application compiled CSS
> - third party CSS?
> - etc ..

File: development/css.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+## How
+
+CSS can be written and maintained in many different ways. Developers at Telus are discouraged from writing CSS directly, and to instead go with the following options in order of preference:
+
+1. Consume [React](./react.md) components from [TDS](http://tds.telus.com/)
+1. Write [custom React components](./react.md#how)
```
Comment (ahmadnassri) [inline development/css.md]: use numbered lists with actual numbers, for readers who may not be viewing HTML output of markdown

**Comment by theetrain** [comment]
@ahmadnassri made more changes. As far as I know, there aren't any vendor CSS used on telus.com, but we have so many properties I decided not to check them all given the time I have - so I kept the explanation purely ideological.

**Comment by ahmadnassri** [comment]
@theetrain I don't assume we have vendor CSS, but perhaps things like Adobe Target?

also, post in Slack, see if some insight emerges!

**Pull request merged**